ecommerce/views.py

```diff
@@ -7,16 +7,17 @@
 
 from django.shortcuts import render
 
-from ecommerce.models import Slider
+from ecommerce.models import Slider,Product, SocailMedia
 
 
 class IndexView(generic.TemplateView):
     template_name = 'index.html'
 
     def get(self, request, *args, **kwargs):
         slider_images = Slider.objects.all()
-
-        return render(request, self.template_name, {'slider_images': slider_images})
+        products = Product.objects.all()
```

Comment:
Please update models and queries.

Product.objects.filter(active=True)
SocailMedia.objects.filter(active=True)